src/sketchup-stl.rb

```diff
@@ -52,7 +52,7 @@ def self.translate(string)
       'This is an open source project sponsored by the SketchUp team. More ' <<
       'info and updates at https://github.com/SketchUp/sketchup-stl'
     )
-    extension.version = '2.1.6'
```

Comment:
I have been meanig to ask - we can squash on our end also before doing the pull request. Do you prefer to see all our small changes, or just the final result in a pull request?